luna/featurevis/relu_grad.py

```diff
@@ -0,0 +1,116 @@
+"""
+Copyright [2017] [https://github.com/tensorflow/lucid]
+"""
```

Comment:
Is that really a 1:1 copy? I would assume we'd have to change some things to make it work with tf2. If that is the case, we should probably mention that we adopted but modified the code from there.

Reply:
More importantly, I don't see any place where we are using these functions. If we are not using them, we should remove this file.